### PR TITLE
test case for validating incorrect content type

### PR DESCRIPTION
**Pull request by shavantha**
this test case validates if the content type is different to application/json. We are passing application/xml and then assert for error code 406 on the response 

tested environment
1.ubuntu 18.04
2.java 1.8.0_161

### CODE REVIEW

**Comment by shavantha** [comment]
closing the PR until the cleanup

File: product-scenarios/1-manage-users-roles/1.1-user-registration-to-an-application/1.1.1-user-registration-with-web-application-itself/1.1.1.1-provision-user-using-SCIM1.1/src/test/java/org/wso2/identity/scenarios/test/scim/UserProvisionSCIMWithDifferentContentTypeTestCase.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        rootObject.put(SCIMConstants.PASSWORD_ATTRIBUTE, SCIMConstants.PASSWORD);
+
+        HttpResponse response = sendPostRequestWithJSON(client, scimUsersEndpoint, rootObject,
+                new Header[]{getBasicAuthzHeader(), getContentTypeApplicationXMLHeader()});
+
+        assertEquals(response.getStatusLine().getStatusCode(), HttpStatus.SC_NOT_ACCEPTABLE, "User has not been created successfully");
+        JSONObject responseObj = getJSONFromResponse(response);
+        responseMessage = responseObj.toJSONString();
+
+        assertEquals(responseMessage,"{\"Errors\":[{\"code\":\"406\",\"description\":\"Requested format is not supported.\"}]}");
```
Comment (ayshsandu) [inline product-scenarios/1-manage-users-roles/1.1-user-registration-to-an-application/1.1.1-user-registration-with-web-application-itself/1.1.1.1-provision-user-using-SCIM1.1/src/test/java/org/wso2/identity/scenarios/test/scim/UserProvisionSCIMWithDifferentContentTypeTestCase.java]: shall we assert code and description separately?

File: product-scenarios/1-manage-users-roles/1.1-user-registration-to-an-application/1.1.1-user-registration-with-web-application-itself/1.1.1.1-provision-user-using-SCIM1.1/src/test/java/org/wso2/identity/scenarios/test/scim/UserProvisionSCIMWithDifferentContentTypeTestCase.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    }
+
+    private Header getBasicAuthzHeader() {
+
+        return new BasicHeader(HttpHeaders.AUTHORIZATION, constructBasicAuthzHeader(ADMIN_USERNAME, ADMIN_PASSWORD));
+    }
+
+    private Header getContentTypeApplicationXMLHeader() {
+
+        return new BasicHeader(HttpHeaders.CONTENT_TYPE, "application/xml");
```
Comment (ayshsandu) [inline product-scenarios/1-manage-users-roles/1.1-user-registration-to-an-application/1.1.1-user-registration-with-web-application-itself/1.1.1.1-provision-user-using-SCIM1.1/src/test/java/org/wso2/identity/scenarios/test/scim/UserProvisionSCIMWithDifferentContentTypeTestCase.java]: "application/xml" move to constant

Comment (shavantha) [reply product-scenarios/1-manage-users-roles/1.1-user-registration-to-an-application/1.1.1-user-registration-with-web-application-itself/1.1.1.1-provision-user-using-SCIM1.1/src/test/java/org/wso2/identity/scenarios/test/scim/UserProvisionSCIMWithDifferentContentTypeTestCase.java]: done, the variable was moved as a constant

File: product-scenarios/1-manage-users-roles/1.1-user-registration-to-an-application/1.1.1-user-registration-with-web-application-itself/1.1.1.1-provision-user-using-SCIM1.1/src/test/java/org/wso2/identity/scenarios/test/scim/UserProvisionSCIMWithDifferentContentTypeTestCase.java (hunk 1 above)
Comment (shavantha) [reply product-scenarios/1-manage-users-roles/1.1-user-registration-to-an-application/1.1.1-user-registration-with-web-application-itself/1.1.1.1-provision-user-using-SCIM1.1/src/test/java/org/wso2/identity/scenarios/test/scim/UserProvisionSCIMWithDifferentContentTypeTestCase.java]: added seperatly

**Comment by shavantha** [comment]
closing this PR as a wrong file has been added "ProvisionUserWithAllAttributesTestCase.java"

**Pull request closed without merging**